drivers/include/net/netdev2.h

```diff
@@ -128,7 +128,7 @@ typedef struct netdev2_driver {
      *
      * @return nr of bytes sent, or <=0 on error
      */
-    int (*send)(netdev2_t *dev, const struct iovec *vector, int count);
```

Comment:
I can also use `unsigned` here, because it's not really a size. I think there is no reason to stay with POSIX here. A wrapper can cast if ever want to give `readv` access to `netdev2` directly. Since the range of `int` `>0` is smaller than the one of `unsigned` or `size_t` we should be on the save side.